python/beeai_framework/agents/react/runners/default/runner.py

```diff
@@ -192,7 +192,7 @@ async def on_new_token(data: dict[str, Any], event: EventMeta) -> None:
                 messages=self.memory.messages[:],
                 stream=True,
                 tools=self._input.tools if self.use_native_tool_calling else None,
-            ).observe(lambda llm_emitter: llm_emitter.on("newToken", on_new_token))
+            ).observe(lambda llm_emitter: llm_emitter.on("new_token", on_new_token))
```

Comment:
same

Reply:
I'll document this in the commit message